### PR TITLE
Introduced Native CCH router

### LINKED ISSUE #3292: CCH: check and prepare CCH PR to be merged into develop
There is PR https://github.com/LBNL-UCB-STI/beam/pull/3120
It was reviewed twice, but it is big, contains many changes and author probably too busy to finish with the PR.

One needs to review it third time in order to be sure the PR is not changing existing behavior and only adds new (i.e. other routers)

### PR DESCRIPTION
**Pull request by dimaopen**
This change is [Reviewable](https://reviewable.io/reviews/lbnl-ucb-sti/beam/3307)

### CODE REVIEW

**Comment by dimaopen** [comment]
Test!

**Comment by dimaopen** [comment]
Test!

File: src/main/scala/beam/router/ParquetRequester.scala [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    val (_, cfg) = prepareConfig(args, isConfigArgRequired = true)
+    val workerParams: R5Parameters = R5Parameters.fromConfig(cfg)
+    val requester = new CchRouteRequester(workerParams, new FreeFlowTravelTime)
+
+    val portionSize = 500000
+    val counter = new AtomicInteger(0)
+    val totalCounter = new AtomicInteger(0)
+    var i = 0
+    do {
+      val requests = getRequests(i * portionSize, portionSize, "/home/crixal/Downloads/0.routingRequest.parquet")
```
Comment (alex-vv) [inline src/main/scala/beam/router/ParquetRequester.scala]: hardcoded local path

**Comment by dimaopen** [comment]
Test now!

**Review by alex-vv — APPROVED** [review]

**Pull request merged**